Sources/SwiftProtobuf/Message.swift

```diff
@@ -90,16 +94,12 @@ public protocol Message: CustomDebugStringConvertible {
                                                extensions: ExtensionSet?) throws
 
   //
-  // Protobuf Text decoding
-  //
-  mutating func decodeTextFormat(from: inout TextFormatDecoder) throws
-
-  //
   // google.protobuf.Any support
   //
 
-  // Decode from an `Any` (which might itself have been decoded from JSON,
-  // protobuf, or another `Any`).
+  /// Decode this object from an `Any` (which might itself have been
+  /// decoded from JSON, protobuf, or another `Any`).  This is the
+  /// analog of `unpack` in the Google C++ or Java API.
   init(any: Google_Protobuf_Any) throws
```

Comment:
Good idea!  This initializer is now in Google_Protobuf_Any as a general Message extension (but never needs to be overridden).  The encoding is now hidden within Google_Protobuf_Any.

Question:  Should this initializer be `init(unpacking:)` instead of `init(any:)`?  I noticed in updating this that Value also has an `init(any:)` that takes a `Swift.Any` and it seems that might be confusing.